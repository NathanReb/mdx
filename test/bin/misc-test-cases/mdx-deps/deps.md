Toto

```ocaml file=tikitaka.ml
```
  |}
  and lines2 = Mdx.of_string Mdx.Normal
  {|
Tata

```ocaml file=tuktuk.ml,skip
```

```cram file=burn.sh
```

```sh dir=ping/
```
