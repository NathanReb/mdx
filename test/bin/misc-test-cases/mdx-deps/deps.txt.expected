deps.md: file:tikitaka.ml file:burn.sh dir:ping/
deps2.md: file:tikitakabis.ml dir:ping/pong file:foo/bar.txt file:burn.sh
